python-backend/app/api/permits/response_models.py

```diff
@@ -0,0 +1,49 @@
+from app.extensions import api
+from flask_restplus import fields
+
+PERMIT_AMENDMENT_DOCUMENT_MODEL = api.model(
```

Comment:
I think given how many things now live under the mine we may want to start breaking the API models up into separate files like this instead of having one large model file.

Reply:
I agree.